src/components/base/Modal/ConfirmModal.js

```diff
@@ -52,13 +53,14 @@ class ConfirmModal extends PureComponent<Props> {
       analyticsName,
       centered,
       children,
+      narrow,
       ...props
     } = this.props
 
     const realConfirmText = confirmText || t('common.confirm')
     const realCancelText = cancelText || t('common.cancel')
     return (
-      <Modal isOpened={isOpened} centered={centered}>
+      <Modal isOpened={isOpened} centered={centered} width={narrow && 380}>
```

Comment:
how did this pass flow?

narrow can be a `boolean`, so it can be `false`. width however takes a `?: number` not a `?: number | boolean` . 

I would suggest to do `narrow ? 380 : undefined` , less fancy but usually better for type safety

Reply:
oh my